src/main/scala/is/hail/stats/package.scala

```diff
@@ -5,7 +5,7 @@ import is.hail.annotations.Annotation
 import is.hail.utils._
 import is.hail.variant.{Genotype, Variant, VariantDataset, VariantMetadata}
 import net.sourceforge.jdistlib.{ChiSquare, Normal}
-import org.apache.commons.math3.distribution.HypergeometricDistribution
+import org.apache.commons.math3.distribution.{HypergeometricDistribution, PoissonDistribution}
```

Comment:
Use the jdistlib implementations rather than Apache commons.  They are replicas of the C code underlying R, so fast and, most importantly, handle extreme numerical issues better.